### ISSUE TITLE
Consistent doc header casing

### ISSUE THREAD

**Issue by johanandren** [akka/akka#27741]
Noted in https://github.com/akka/akka/pull/27708#discussion_r325227829 that our uppercasing in headers is all over the place, we should pick one strategy and stick with it.

My suggestion, `Title Capitalization For The Page Title` (h1) and `Normal capitalization for the subheaders`. (I have no writing practice knowledge to motivate it with though)

**Comment by johanandren** [comment]
Recommendation from someone with better know how: no title case at all. 

Not important enough that we'll spend time on it though. If someone has a lot of free time, it's up to them what they do with it. ;)

**Comment by helena** [comment]
Note however that some things may need capitalization, .e.g.

# Header for Cluster thingy

or

# Header for cluster thingy

**Comment by johanandren** [comment]
Yeah, the names will always be capitalized, like Cluster Sharding, right?

**Comment by helena** [comment]
Ya, perfect example.

**Comment by patriknw** [comment]
yes, "product" or "feature" names should be title case, Cluster Sharding or `ClusterSharding` if referring to the actual class.
